docker/mysql/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mysql:8.0.17
+ENV MYSQL_ROOT_PASSWORD root
+ENV MYSQL_USER dbuser
+ENV MYSQL_PASSWORD dbpassword
```

Comment:
Credentials shouldn't be included in Dockerfile. They should be provided only at runtime.